searchtool/server/views/layout.hbs

```diff
@@ -183,7 +183,8 @@
                           {{#if term}}
                             <button aria-expanded="false" aria-haspopup="true" data-toggle="dropdown" class="btn btn-default dropdown-toggle" id="btn-pgActions">Download <span class="caret"></span></button>
                             <ul aria-labelledby="btn-pgActions" class="dropdown-menu">
-                              <li role="presentation"><a target="_BLANK" href="/download?q={{term}}&dataset={{dataset}}&fromdate={{fromdate}}&todate={{todate}}&art_unit={{artunit}}&documentcode={{documentcode}}">Download CSV</a></li>
+                              <li role="presentation"><a target="_BLANK" href="/download?q={{term}}&dataset={{dataset}}&fromdate={{fromdate}}&todate={{todate}}&art_unit={{artunit}}&documentcode={{documentcode}}&textdata=yes">CSV with text</a></li>
```

Comment:
True but if it is more than 2000 and they click on full text, would that break it? My concern is mostly hiding that one.